main.go

```diff
@@ -42,6 +43,7 @@ var opts struct {
 	PagerDutyScheduleEntryTimeFormat   string        `long:"pagerduty.schedule.entry-timeformat"           env:"PAGERDUTY_SCHEDULE_ENTRY_TIMEFORMAT"          description:"PagerDuty schedule entry time format (label)" default:"Mon, 02 Jan 15:04 MST"`
 	PagerDutyIncidentTimeFormat        string        `long:"pagerduty.incident.timeformat"                      env:"PAGERDUTY_INCIDENT_TIMEFORMAT"                description:"PagerDuty incident time format (label)" default:"Mon, 02 Jan 15:04 MST"`
 	PagerDutyDisableTeams              bool          `long:"pagerduty.disable-teams"                description:"Set to true to disable checking PagerDuty teams (for plans that don't include it)"                env:"PAGERDUTY_DISABLE_TEAMS"`
+	PagerDutyTeamListOpt			   []string      `long:"pagerduty.team-listopt"                 env:"PAGERDUTY_TEAM_LISTOPT"            description:"Passes team ID as a list option when applicable."`
```

Comment:
`PagerDutyTeamFilter` and `pagerduty.teamfilter` also here?

Reply:
Use you can use `env-delim:" "` or `env-delim:","` for the delimiter, so you don't need to split it when using env vars